src/modules/ie-objects/elasticsearch/queryBuilder.ts

```diff
@@ -228,6 +230,7 @@ export class QueryBuilder {
 							},
 						},
 					],
+					minimum_should_match: 2,
```

Comment:
REQ: fix test cases for QueryBuilder

4 test cases are failing because of these changes please fix.

<img width="1011" alt="Screenshot 2023-03-10 at 20 02 16" src="https://user-images.githubusercontent.com/37472626/224404064-86cada43-56bd-471c-90a2-3c3889449a40.png">
<img width="793" alt="Screenshot 2023-03-10 at 20 02 36" src="https://user-images.githubusercontent.com/37472626/224404065-3fd01296-ff79-474e-a7e3-a5d7ed6cc6f6.png">
<img width="784" alt="Screenshot 2023-03-10 at 20 02 43" src="https://user-images.githubusercontent.com/37472626/224404068-897b3153-e295-4c84-bd77-7256235f7c19.png">
<img width="1009" alt="Screenshot 2023-03-10 at 20 02 52" src="https://user-images.githubusercontent.com/37472626/224404071-82fc4008-2b38-4aba-8147-ba7f74dd2a86.png">
<img width="773" alt="Screenshot 2023-03-10 at 20 03 01" src="https://user-images.githubusercontent.com/37472626/224404073-a169735b-993a-45c6-abf9-fa69728e311f.png">